docker-for-mac/osxfs-caching.md

```diff
@@ -0,0 +1,195 @@
+---
+description: OSXFS CACHING
```

Comment:
Is this description intended to be in ALL CAPS?  The other descriptions in the directory are in normal prose.